warehouse/core/src/main/java/datawave/util/CompositeTimestamp.java

```diff
@@ -0,0 +1,42 @@
+package datawave.util;
+
+public class CompositeTimestamp {
```

Comment:
A class description would be good that describes how the long value is split into two values.